docs/zero_to_hero_guide/quickstart.md

```diff
@@ -0,0 +1,183 @@
+# Llama Stack Quickstart Guide
+
+This guide will walk you through setting up an end-to-end workflow with Llama Stack, enabling you to perform text generation using the `Llama3.1-8B-Instruct` model. Follow these steps to get started quickly.
+
+If you're looking for more specific topics like tool calling or agent setup, we have a [Zero to Hero Guide](#next-steps) that covers everything from Tool Calling to Agents in detail. Feel free to skip to the end to explore the advanced topics you're interested in.
+
+## Table of Contents
+1. [Prerequisite](#prerequisite)
+2. [Installation](#installation)
+3. [Download Llama Models](#download-llama-models)
+4. [Build, Configure, and Run Llama Stack](#build-configure-and-run-llama-stack)
+5. [Testing with `curl`](#testing-with-curl)
+6. [Testing with Python](#testing-with-python)
+7. [Next Steps](#next-steps)
+
+---
+
+## Prerequisite
+
+Ensure you have the following installed on your system:
+
+- **Conda**: A package, dependency, and environment management tool.
+
+---
+
+## Installation
+
+The `llama` CLI tool helps you manage the Llama Stack toolchain and agent systems.
+
+**Install via PyPI:**
+
+```bash
+pip install llama-stack
+```
+
+*After installation, the `llama` command should be available in your PATH.*
+
+---
+
+## Download Llama Models
+
+Download the necessary Llama model checkpoints using the `llama` CLI:
+
+```bash
+llama download --model-id Llama3.1-8B-Instruct
+```
+
+*Follow the CLI prompts to complete the download. You may need to accept a license agreement. Obtain an instant license [here](https://www.llama.com/llama-downloads/).*
+
+---
+
+## Build, Configure, and Run Llama Stack
+
+### 1. Build the Llama Stack Distribution
+
+We will default into building a `meta-reference-gpu` distribution, however you could read more about the different distriubtion [here](https://llama-stack.readthedocs.io/en/latest/getting_started/index.html#decide-your-inference-provider).
```

Comment:
typo: `distributions`

Reply:
I think we could expand this into a couple sentences maybe. At least tell the user why we are using this distribution?

Reply:
yep, elaborated and also mentioned about other distro options we offer.